src/index.js

```diff
@@ -43,9 +42,15 @@ const DataSelectorFinder = require('./domain/common/DataSelectorFinder');
 
 const BlockListener = require('./infrastructure/blockchain/BlockListener');
 
+const { RequestRepositoryFactory, ResponseRepositoryFactory } = require('./infrastructure/persistance');
+
+if (process.env.PERSISTANCE === 'MONGODB') {
```

Comment:
1. Should we reuse the constant from enum?
1. This could be some kind of a separate object, so that mongoose doesn't leak into index.js